src/rollup-vsix-plugin.ts

```diff
@@ -96,19 +96,27 @@ export default function plugin (options: Options = defaultOptions): Plugin {
       const files = await readVsix(id)
       const manifest = JSON.parse(files['package.json']!.toString('utf8'))
 
-      const usedFiles = ['package.json', ...extractPathsFromExtensionManifest(manifest.contributes)]
+      const usedFiles = ['package.json', 'package.nls.json', ...extractPathsFromExtensionManifest(manifest.contributes)]
+      const nlsExists = files['package.nls.json'] != null
 
-      const vsixFile: Record<string, Buffer> = usedFiles.reduce((acc, usedFile) => ({
-        ...acc,
-        [usedFile]: files[path.relative('/', path.resolve('/', usedFile))]!
-      }), {} as Record<string, Buffer>)
+      const vsixFile: Record<string, Buffer> = usedFiles.reduce((acc, usedFile) => {
+        const filePath = files[path.relative('/', path.resolve('/', usedFile))]
```

Comment:
Does it make sense to use Platform specific path separators like `path.sep`?

Reply:
I've published a 1.78.3-next.0

> Does it make sense to use Platform specific path separators like path.sep?

I don't think so... it's just some virtual internal paths

Reply:
Ok.